packages/ui-library/src/components/input-field-number/index.stories.ts

```diff
@@ -607,6 +607,7 @@ export const HasError = (params: BlrInputFieldNumberType) => {
 
 /**
  * ## Dependencies
+ 
```

Comment:
Please unify these new lines for consistency.

Reply:
done